examples/jsm/renderers/webgpu/WebGPUTextures.js

```diff
@@ -51,6 +52,26 @@ class WebGPUTextures {
 
 	}
 
+	getVideoDefaultTexture() {
+
+		if ( this.defaultVideoTexture === null ) {
+
+			const video = document.getElementById( 'video' );
```

Comment:
I think we should do something like this here:
```js
const video = typeof document !== 'undefined' ? document.createElement( 'video' ) : null;
```

Reply:
Hmm... My first attempt was just put `THREE.VideoTexture()` but it crashes here:
https://github.com/mrdoob/three.js/blob/2745552562cb9d99d04c619de56820f571020eac/src/textures/VideoTexture.js#L26

and maybe should crashes using `device.importExternalTexture()` too without `<video>`.